packages/framework/presence/src/latestValueManager.ts

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { ConnectedClientId } from "./baseTypes.js";
+import type { LatestValueControls } from "./latestValueControls.js";
+import type { LatestValueClientData, LatestValueData } from "./latestValueTypes.js";
+
+import type {
+	JsonDeserialized,
+	JsonSerializable,
+} from "@fluid-experimental/presence/internal/core-interfaces";
+import type { ISubscribable } from "@fluid-experimental/presence/internal/events";
+import type { InternalTypes } from "@fluid-experimental/presence/internal/exposedInternalTypes";
+import type { InternalUtilityTypes } from "@fluid-experimental/presence/internal/exposedUtilityTypes";
+
+/**
+ * @sealed
+ * @alpha
+ */
+export interface LatestValueManagerEvents<T> {
+	/**
+	 * Raised when remote client's value is updated, which may be the same value.
+	 *
+	 * @eventProperty
+	 */
+	updated: (update: LatestValueClientData<T>) => void;
+}
+
+/**
+ * Value manager that provides the latest known value from this client to others and read access to their values.
+ * All participant clients must provide a value.
+ *
+ * @remarks Create using {@link Latest} registered to {@link PresenceStates}.
+ *
+ * @sealed
+ * @alpha
+ */
+export interface LatestValueManager<T> {
+	/**
+	 * Events for Latest value manager.
+	 */
+	readonly events: ISubscribable<LatestValueManagerEvents<T>>;
+
+	/**
+	 * Controls for management of sending updates.
+	 */
+	readonly controls: LatestValueControls;
```

Comment:
That is the intention. The values can be read and manipulated. Implementation used property accessors for `forcedRefreshInterval` since we'll need to know when there is a change. `allowableUpdateLatency` is used passively, currently.